lib/v2/datasource/remote/model/profile_model.dart

```diff
@@ -1,28 +1,28 @@
 class ProfileModel {
-  final String? account;
-  final String? status;
-  final String? type;
-  final String? nickname;
-  final String? image;
-  final String? story;
-  final String? roles;
-  final String? skills;
-  final String? interests;
-  final int? reputation;
-  final int? timestamp;
+  final String account;
+  final String status;
+  final String type;
+  final String nickname;
+  final String image;
+  final String story;
+  final String roles;
+  final String skills;
+  final String interests;
+  final int reputation;
+  final int timestamp;
 
   ProfileModel({
-    this.account,
-    this.status,
-    this.type,
-    this.nickname,
-    this.image,
-    this.story,
-    this.roles,
```

Comment:
is this required?

Reply:
please make sure the fields marked as required are truly required

Reply:
Yes,  all of these fields come from back-end.

Reply:
so, whenever we are going to use the imageUrl, we need to check if the imageUrl is empty first, before using it. right ?

Reply:
I think the backend should not be sending default empty strings, 
But since they are doing that, we should just turn empty strings into nulls in our models.

Reply:
In that case i think only account and timestamp would be no-nullable the rest values can be null, I'm not sure how is a default account when is created if it also has and status and type defaults .. mmm

Reply:
I added the handle functions for the mapping from JSON